src/main/scala/org/labrad/manager/Listener.scala

```diff
@@ -12,42 +14,65 @@ import scala.concurrent.ExecutionContext
 /**
```

Comment:
In other files it seems you have 2 lines between imports and the first class and here only one - not sure which is preferred